Energy Saver/Pages/Index.cshtml

```diff
@@ -32,7 +32,21 @@
                 @foreach (var taxRecord in year)
                 {
                     <tr>
-                        <td style="text-align:left">@taxRecord.Month.ToString()</td>
+                        <td>
+                            <div class="row">
+                                <div style="text-align:left; font-weight: bold" class="col-2">
+                                    @taxRecord.Month.ToString()
+                                </div>
+                                <div style="text-align:right" class="col">
+                                    <div class="row toEvaluate">
+                                        <div class="col">
+                                            <i class="icon"></i>
+                                        </div>
```

Comment:
Maybe there should be a table header called Tax average or something similar. Now it seems unclear what the percentage stands for